### PR TITLE
Cryogenic freezers change

### PR DESCRIPTION
**Pull request by Redory**
## About The Pull Request

Big changes and my first "big" pr so please testmerge thank

Completely changes the cryopods in the game to instead of wiping you from the face of existence to instead just preserve your body with godmode so you can come back to the round without being dead.

You will need at least a cryopod and an oversight control console in the same area for it to work. Both printable after researching the cryo techweb thingy for 5000 good boi points (was 2000)

Oversight console can turn yellow if a person decides to go into cryo with certain high value items like caps spare, blueprints or budget cards on em.

New tgui for the oversight control console to be able to eject people.
(Ejecting everyone from the console at once or ejecting the last person doesn't update the tgui to show it empty)

If you are a slimeperson with clones, your consciousness will NOT transfer into a clone in cryo.

Made certain damage procs respect the GODMODE flag that mobs can have.
GODMODE'd players are bomb proof and all items on the player are safe from explosions, acid, fire, etc.
GODMODE change needed so players don't die to bullshit while in cryo.

You wake up in cryo:
grafik

When going into cryo it tells you an ETA:
grafik

When in cryo, click this to eject yourself:
grafik

Fancy tgui, also supports silicons:
grafik

Ejected the occupants (might need to close and open tgui again to refresh):
grafik

Techweb integration in existing cryo tech:
grafik

console turns yellow if a person takes a high value item with them into cryo:
grafik

Fun fact!
Everyone is being stuffed into the cryo console itself so if any shenanigans happen it happens locally and if someone gets stuck in there somehow, check the console's /contents.

## Why It's Good For The Game

Getting wiped from existence bad on a 24/7 serb.
Preserve bodies to not get fucked by anomalies 24/7 good.

## Changelog
:cl:
add: New Cryopods which don't delete you with fancy tgui
del: Old Cryopods
balance: made godmoded mobs be explosion immune and also makes all their belongings immune to damage too
imageadd: new cryopod console state (yellow)
code: used defines in some explosion code instead of numerals
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 1]
```diff
@@ -384,6 +384,8 @@
 
 
 /mob/living/carbon/human/ex_act(severity, target, origin)
+	if(status_flags & GODMODE) // SinguloStation13 Edit (Cryogenic freezers change - Explosion proof if Godmode is enabled)
```
Comment (Urumasi) [inline code/modules/mob/living/carbon/human/human_defense.dm:387]: Is this check actually required if it's also performed in /mob/living/?

Comment (Redory) [reply code/modules/mob/living/carbon/human/human_defense.dm:387]: Nothing is being returned upwards on ex_act unlike what emp_act does, so yes it is needed. Just tested it.
I changed emp_act after looking at their procs further. commit comin soon

**Comment by Redory** [comment]
Integration Test fail by bad ruin spawning. Unrelated to my changes.

**Comment by MaltVinegar** [comment]
Based

**Review by MaltVinegar — APPROVED** [review]
Can't find any bugs

**Review by MaltVinegar — APPROVED** [review]

**Pull request merged**